context.go

```diff
@@ -83,6 +86,16 @@ func (c *Context) reset() {
 	}
 }
 
+// SetOtelAttributes sets the provided OpenTelemetry attributes.
```

Comment:
```suggestion
// SetOTelAttributes sets the provided OpenTelemetry attributes.
```

Nit, OpenTelemetry is commonly abbreviated OTel with a capital T.

Reply:
:+1: